vm/chef/cookbooks/mattermost/files/mattermost-setup

```diff
@@ -42,3 +42,8 @@ setup_nginx "${default_domain}"
 
 # Start Mattermost
 systemctl start mattermost
+
+# Wait until ready.
+until curl "http://localhost:80"; do
+  sleep 3
+done
```

Comment:
Can we add a max number of retries here?

Reply:
@EugeneKorolevich please, add something similar to this: https://github.com/GoogleCloudPlatform/click-to-deploy/blob/55a58a6f79250bd00b04bcf74f2d66d5ebf40260/vm/chef/cookbooks/pimcore/files/pimcore#L31